### ISSUE TITLE
Error message dialog if creating main controller failed

### ISSUE THREAD

**Issue by krystiankaluzny** [roskenet/springboot-javafx-support#18]
I have MainWindowController which control MainWindowView. 
MainWindowView class is an argument of launchApp static method.

```
@FXMLController
public class MainWindowController
{
	@PostConstruct
	private void initBean()
	{
            doSomething()
	}

	@FXML
	public void initialize()
	{
	     doSomething2()
	}
}
```
If *doSomething* or *doSomething2* throw RuntimeException then application stuck on the SplashScreen.

The first case is caused by **SpringApplication.run(this.getClass(), savedArgs)** failed in *init()*.
The second is caused by **view.getView()** failed in *showView(Class<? extends AbstractFxmlView> newView)*

I think that it would by more informative to show some error dialog or even close the app.

**Comment by roskenet** [comment]
Yes, you are right. I'm thinking all the time about the "hanging splash screen".
Will address this soon.

**Comment by roskenet** [comment]
Fixed with 1.3.23
Next step: Perhaps a i18n configurable text. (Blah, blah... Please send the log file to helpdesk@company.com)
